VERSION

```diff
@@ -0,0 +1 @@
+0.1.0
```

Comment:
What is this file used for?

Reply:
Define Potsie's release (as for Arnold's project), since I haven't found a better way to do so for jsonnet-based projects.